tests/unit/hmrc/mtdFraudPrevention.test.js

```diff
@@ -93,6 +102,9 @@ describe("FraudPreventionHeaders", () => {
     );
     expect(headers.get("Gov-Client-Browser-Do-Not-Track")).toBe("true");
     expect(headers.get("Gov-Client-Local-IPs")).toBe(undefined);
+    expect(headers.get("Gov-Client-Device-ID")).toBe(
+        "uuid=424f48aa-b723-4f97-8a30-d214b43bf372"
```

Comment:
As per the issue, the expected format of these headers is `Gov-Client-Device-ID: beec798b-b366-47fa-b1f8-92cede14a1ce`

Please refer to the documentation on how to run your tests locally

Reply:
updated the code to reflect the changes